docs/ionq/calibrations.md

```diff
@@ -0,0 +1,60 @@
+# IonQ API Calibrations
+
+Calibrations are snapshots of the performance of the IonQ platform
+at a given moment of time.  The IonQ quantum computers are continuously
+calibrated, and periodically measurements are taken of the performance
```

Comment:
`and periodically measurements are taken of the performance
of the machine.` -> `and measurements are taken of the performance
of the machine periodically`

Reply:
done